release/7-2/mariner2/docker/Dockerfile

```diff
@@ -0,0 +1,87 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+FROM cblmariner2preview.azurecr.io/base/core:2.0 AS installer-env
+
+# Define Args for the needed to add the package
+ARG PS_VERSION=7.3.0-preview.1
```

Comment:
shouldn't this be version 7.2 if it's in the 7-2 folder?

Reply:
```suggestion
ARG PS_VERSION=7.2.3
```